Assets/Scripts/Internal/DaggerfallBillboard.cs

```diff
@@ -439,6 +439,27 @@ public void OnDrawGizmos()
                 }
             }
         }
+        
+        /// <summary>
+        /// Draws a circle at the bottom of the bilboard to make it easier to judge the size regardless of rotation.
+        /// </summary>
+        public void OnDrawGizmosSelected()
+        {
+            if (!Application.isPlaying)
+            {
+                UnityEditor.SceneView sceneView = GetActiveSceneView();
+                if (sceneView)
```

Comment:
Hi. You don't need to test `sceneView!=null`, it's UnityEditor's responsibility to decide on that when `OnDrawGizmos` is called.

Reply:
Oh, ok - I just looked into the the code before your change and saw that this test was there before. I still think this line is counter-productive, no reasons for it and can be deleted (might be a legacy code, from much older editor version maybe).

Reply:
Removed.